content/en/docs/collector/getting-started.md

```diff
@@ -1,358 +1,90 @@
 ---
 title: Getting Started
 # prettier-ignore
-cSpell:ignore: darwin dpkg GOARCH journalctl kubectl otelcorecol pprof tlsv zpages
+cSpell:ignore: otelcol telemetrygen docker jaeger zipkin prometheus dpkg
 weight: 1
 ---
 
-If you aren't familiar with the deployment models, components, and repositories
-applicable to the OpenTelemetry Collector, first review the [Data Collection][]
-and [Deployment Methods][] page.
+The OpenTelemetry Collector listens to metrics, logs, and traces, processes the
+telemetry, and exports it to a wide variety of observability back-ends using its
+components. For a conceptual overview of the Collector, see [Collector][].
 
-## Demo
+The following tutorial shows how to deploy the Collector in agent mode and send
```

Comment:
```suggestion
The following tutorial shows how to deploy the Collector and send
```

I think at this place the deployment pattern is not relevant and leads to more confusion